packages/flutter_tools/test/commands.shard/hermetic/build_macos_test.dart

```diff
@@ -112,7 +112,7 @@ void main() {
     createCoreMockProjectFiles();
 
     expect(createTestCommandRunner(command).run(
-      const <String>['build', 'macos']
+      const <String>['build', 'macos', '--no-pub']
```

Comment:
Oh hey, this was another bug we needed to fix

Reply:
I can't find the original issue, but we did not correctly run pub when building for macos

Reply:
https://github.com/flutter/flutter/issues/46236